### PR TITLE
feat(Gamut): tweaked AccordionButton to look more like the nice LE yellow ones

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
## Tweaked AccordionButton to look more like the nice LE yellow ones

* Removed the yellow theme's left and right borders (since the LE doesn't have them)
* Made the arrow rotate into position instead of flip - and a little more quickly

Mike Kenny indicated a lack of preference for which animation is used, and I like the rotate-y one more.

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.8.24-alpha.cf7bce.0<br/>@codecademy/gamut@3.5.1-alpha.cf7bce.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5e97428befb15b6602ce872d--codecademy-styleguide.netlify.app>https://5e97428befb15b6602ce872d--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5e97428befb15b6602ce872d>Deploy Logs</a></p>

File: packages/gamut/src/AccordionButton/styles.module.scss [hunk 1]
```diff
@@ -29,9 +30,9 @@
 }
 
 .expansionIcon {
-  transition: transform 0.5s;
+  transition: transform 0.35s ease-out;
 }
 
 .expansionIconExpanded {
-  transform: rotateX(180deg);
+  transform: rotate(-180deg);
```
Comment (codecaaron) [inline packages/gamut/src/AccordionButton/styles.module.scss:37]: Do we want to be explicit that this is on the Z axis?

Comment (JoshuaKGoldberg) [reply packages/gamut/src/AccordionButton/styles.module.scss:37]: Sorry, I'm pretty out of the loop on how CSS refers to axes 😄  - but changing to this switched from a flip to a rotate. Is there a better way to make it @codecaaron?

Comment (JoshuaKGoldberg) [reply packages/gamut/src/AccordionButton/styles.module.scss:37]: @codecaaron I'm gonna merge now to unblock some other work but down to improve this later on 💪

**Pull request merged**